app/models/overlay/member.rb

```diff
@@ -16,7 +16,7 @@ class Member
 
       # There can be at most one of this types per visualization
       UNIQUE_TYPES = [
-          'header', 'search', 'layer_selector', 'share', 'zoom', 'logo', 'loader', 'fullscreen'
```

Comment:
Exactly -- it was throwing an error when the frontend attempted to create the second header.